scripts/test-react-native-setup.sh

```diff
@@ -2,12 +2,16 @@
 
 set -e
 
-echo "Check Buck setup"
+export KOTLIN_HOME="ReactAndroid/src/main/third-party/kotlin"
```

Comment:
I think you need to have KOTLIN_HOME as `third-party/kotlin` instead, after latest changes